examples/AnnotatorForClinicalDataExample.java

```diff
@@ -0,0 +1,543 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import com.ibm.cloud.sdk.core.http.Response;
+import com.ibm.cloud.sdk.core.http.ServiceCall;
+import com.ibm.cloud.sdk.core.security.Authenticator;
+import com.ibm.cloud.sdk.core.security.BasicAuthenticator;
+import com.ibm.watson.health.acd.v1.AnnotatorForClinicalData;
+import com.ibm.watson.health.acd.v1.model.AcdCartridges;
+import com.ibm.watson.health.acd.v1.model.AcdFlow;
+import com.ibm.watson.health.acd.v1.model.AcdProfile;
+import com.ibm.watson.health.acd.v1.model.Annotator;
+import com.ibm.watson.health.acd.v1.model.Annotator.Name;
+import com.ibm.watson.health.acd.v1.model.AnnotatorFlow;
+import com.ibm.watson.health.acd.v1.model.CartridgesGetIdOptions;
+import com.ibm.watson.health.acd.v1.model.CartridgesGetOptions;
+import com.ibm.watson.health.acd.v1.model.CartridgesPostMultipartOptions;
+import com.ibm.watson.health.acd.v1.model.CartridgesPutMultipartOptions;
+import com.ibm.watson.health.acd.v1.model.Concept;
+import com.ibm.watson.health.acd.v1.model.ContainerGroup;
+import com.ibm.watson.health.acd.v1.model.CreateFlowsOptions;
+import com.ibm.watson.health.acd.v1.model.CreateProfileOptions;
+import com.ibm.watson.health.acd.v1.model.CreateProfileOptions.Builder;
+import com.ibm.watson.health.acd.v1.model.DeleteFlowsOptions;
+import com.ibm.watson.health.acd.v1.model.DeleteProfileOptions;
+import com.ibm.watson.health.acd.v1.model.DeleteUserSpecificArtifactsOptions;
+import com.ibm.watson.health.acd.v1.model.DeployCartridgeOptions;
+import com.ibm.watson.health.acd.v1.model.DeployCartridgeResponse;
+import com.ibm.watson.health.acd.v1.model.Flow;
+import com.ibm.watson.health.acd.v1.model.FlowEntry;
+import com.ibm.watson.health.acd.v1.model.GetAnnotatorsByIdOptions;
+import com.ibm.watson.health.acd.v1.model.GetAnnotatorsOptions;
+import com.ibm.watson.health.acd.v1.model.GetFlowsByIdOptions;
+import com.ibm.watson.health.acd.v1.model.GetFlowsOptions;
+import com.ibm.watson.health.acd.v1.model.GetProfileOptions;
+import com.ibm.watson.health.acd.v1.model.GetProfilesOptions;
+import com.ibm.watson.health.acd.v1.model.ListStringWrapper;
+import com.ibm.watson.health.acd.v1.model.ServiceApiBean;
+import com.ibm.watson.health.acd.v1.model.ServiceStatus;
+import com.ibm.watson.health.acd.v1.model.UpdateFlowsOptions;
+import com.ibm.watson.health.acd.v1.model.UpdateProfileOptions;
+import com.ibm.watson.health.acd.v1.util.FlowUtil;
+
+/**
+ * Sample ACD Java SDK Calls
+ */
```

Comment:
Do we want more verbiage here about the need to replace ACD_APIKEY, ACD_URL, and version variable values?